src/shared/pkcs11-util.c

```diff
@@ -896,13 +897,11 @@ int pkcs11_token_find_private_key(
 
                 switch (attributes[i].type) {
                 case CKA_CLASS: {
-                        CK_OBJECT_CLASS c;
```

Comment:
if you remove all local variables, you can also drop the {} around this case block

Reply:
The local variable is still here.